inc/namespace.php

```diff
@@ -390,3 +393,26 @@ function real_url_path( string $url, string $handle ) : string {
 
 	return $url;
 }
+
+/**
+ * Delete signups object cache before we load the signups page.
+ *
+ * @param object $signup_query The current instance of the WP_Signup_Query object.
+ */
+function altis_pre_get_signups( $signup_query ) {
```

Comment:
We can add type hinting here if we know what the object type is eg:
```suggestion
function altis_pre_get_signups( WP_Signup_Query $signup_query ) {
```
Will need to `use` this class as well though.